go/ct/gen/interval_solver_test.go

```diff
@@ -285,15 +285,12 @@ func TestIntervalSolver_uint64fullrangeAndEdges(t *testing.T) {
 		setup func(*IntervalSolver[uint64])
 		check func(uint64) bool
 	}{
-		"full-range": {
-			setup: func(s *IntervalSolver[uint64]) {},
```

Comment:
Although the `check` does not add any value, running the solver without any constraints is still useful to ensure no error is produced.

Reply:
right.